ddl/ddl_api.go

```diff
@@ -2533,6 +2635,57 @@ func (d *ddl) DropColumn(ctx sessionctx.Context, ti ast.Ident, spec *ast.AlterTa
 	return errors.Trace(err)
 }
 
+// DropColumns will drop multi-columns from the table, now we don't support drop the column with index covered.
+func (d *ddl) DropColumns(ctx sessionctx.Context, ti ast.Ident, specs []*ast.AlterTableSpec) error {
+	schema, t, err := d.getSchemaAndTableByIdent(ctx, ti)
+	if err != nil {
+		return errors.Trace(err)
+	}
+	tblInfo := t.Meta()
+
+	if len(tblInfo.Columns) == len(specs) {
+		return ErrCantRemoveAllFields.GenWithStack("can't drop all columns in table %s",
+			tblInfo.Name)
+	}
+
+	var colNames []model.CIStr
+	for _, spec := range specs {
+		// Check whether dropped column has existed.
+		colName := spec.OldColumnName.Name
```

Comment:
Need check unique name too;

This PR

```sql
mysql>alter table t drop column a,drop column a;
Query OK, 0 rows affected
```

Mysql

```sql
mysql>alter table t drop column a,drop column a;
(1091, u"Can't DROP 'a'; check that column/key exists")
```

Reply:
Hi, should we use the same error like MySQL , or just :
ERROR 1060 (42S21): Duplicate column name 'a'

Reply:
Need use a map here to check unique drop column name, just like `addingColumnNames` map in line 2356.

Reply:
Done.